power_grid_model_c/power_grid_model_c/include/power_grid_model_c.h

```diff
@@ -39,8 +39,13 @@
 // API_MACRO_BLOCK
 
 // integers
+#ifdef __cplusplus
+#include <cstddef>
+#include <cstdint>
+#else
 #include <stddef.h>
 #include <stdint.h>
+#endif
```

Comment:
Do we really need this? What is the down side to include C header in C++?

Reply:
the downside is that everything that is `std` in the `<cstdxxx>` libraries becomes global namespace in `<stdxxx.h>`. that means that users including our C API using C++ code will have their environment polluted with global namespace features. We should aim to keep our users' environment as clean as possible.